man/man1/kbsecret-env.1.ronnpp

```diff
@@ -40,8 +40,8 @@ loading important environment keys into a program without exposing them directly
 	$ kbsecret env -v baz-api
 	thisapiusesapassword
 
-	$ kbsecret env -n baz-api
-	BAZ_API=thisapiusesapassword
+	$ kbsecret env -n -a
```

Comment:
It'd be good to make a separate example for this behavior instead.

Reply:
Fixed.